src/pages/Admin.tsx

```diff
@@ -27,82 +12,13 @@ const Admin = () => {
-                                </TableHead>
-                                <TableBody>
-                                    {connectors.map((row, index) => (
-                                        <TableRow
-                                            key={`Connectors-${index}`}
-                                            sx={{
-                                                '&:last-child td, &:last-child th':
-                                                    { border: 0 },
-                                            }}
-                                        >
-                                            <TableCell
-                                                component="th"
-                                                scope="row"
-                                            >
-                                                {row.attributes.name}
-                                            </TableCell>
-                                            <TableCell>
-                                                {row.attributes.description}
-                                            </TableCell>
-                                            <TableCell>
-                                                {row.attributes.type}
-                                            </TableCell>
-                                            <TableCell>
-                                                {row.attributes.maintainer}
-                                            </TableCell>
-                                            <TableCell>
-                                                <FormattedDate
-                                                    day="numeric"
-                                                    month="long"
-                                                    year="numeric"
-                                                    value={
-                                                        row.attributes
-                                                            .updated_at
-                                                    }
-                                                />
-                                            </TableCell>
-
-                                            <TableCell>
-                                                <a href={row.links.images}>
-                                                    More details
-                                                </a>
-                                            </TableCell>
-                                        </TableRow>
-                                    ))}
-                                </TableBody>
-                            </Table>
-                        </TableContainer>
-                    ) : null}
-                </Box>
+            <Box
+                sx={{
+                    maxHeight: 250,
+                    overflow: 'auto',
+                }}
+            >
+                <ConnectorsTable />
```

Comment:
When including the data table I think we should still include a wrapper so the parent can set the size. Left the header inside the table that way we have consistent language.